### PR TITLE
Navigation menu

### PR DESCRIPTION
**Pull request by gwynethsc**
Adds pop-out sidebar menu

### CODE REVIEW

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -7,6 +7,16 @@
     <script src="script.js"></script>
   </head>
   <body>
+    <div id="navigation-bar">
+        <button id="nav-close" onclick="closeSideNav()">&times;</button>
+        <a href="index.html">Home</a>
+        <a href="cats.html">Cats</a>
+    </div>
+    <div id="navigation-tab" onclick="openSideNav()">
+        <div class="nav-line"></div>
```
Comment (sperzion) [inline portfolio/src/main/webapp/index.html:16]: Are these placeholders?

Comment (gwynethsc) [reply portfolio/src/main/webapp/index.html:16]: These seem to work for me in Chrome, at least; is there nothing visible for you in the top-left corner?

File: portfolio/src/main/webapp/script.js [hunk 2]
```diff
@@ -26,3 +26,19 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+
+/**
+ * Opens side menu
+ */
+function openSideNav() {
+    document.getElementById('navigation-bar').style.display = "inline-flex";
+    document.getElementById('navigation-tab').style.display = "hidden";
```
Comment (Google-JM) [inline portfolio/src/main/webapp/script.js:35]: What you are doing here to change the style works. But a better way would be to move these display changes into you css file in a class, then add and remove the class based on your button click.

Adding or removing classes to your img element on button click.
https://www.w3schools.com/howto/howto_js_add_class.asp
https://www.w3schools.com/howto/howto_js_remove_class.asp

File: portfolio/src/main/webapp/index.html (hunk 1 above)
Comment (sperzion) [reply portfolio/src/main/webapp/index.html:16]: Oh I see. It's the ol' hamburger menu "icon." Cool.

**Pull request merged**